app/Models/Application.php

```
@@ -1301,12 +1303,18 @@ public function generateGitImportCommands(string $deployment_uuid, int $pull_req
                     }
                 } else {
                     $github_access_token = generateGithubInstallationToken($this->source);
```

PHPMD (2.15.0)

[error] The variable $github_access_token is not named in camelCase. (undefined)

(CamelCaseVariableName)